web/packages/shared/components/AccessRequests/AccessDuration/getDurationOptionsFromStartTime.test.ts

```diff
@@ -188,4 +188,6 @@ const mockAccessRequest: AccessRequest = {
   thresholdNames: ['default'],
   resources: [],
   assumeStartTime: null,
+  reasonMode: 'optional',
```

Comment:
i just want to make sure, is this `reasonMode` referring specifically to field `spec.allow.request.reason.mode`

Reply:
It covers both `spec.allow.request.reason.mode` and `spec.options.request_access`

Reply:
I think I know why you may be confused. The information take from the user state may be already outdated. This reasonMode thing will calculated from the current backend state regardless of the roles in the cert.